internal/config/config.go

```diff
@@ -70,7 +83,23 @@ func newEnvironment() environment {
 	vars.BoolVar(&env.NotifyOnCancel, "hellper_notify_on_cancel", true, "Notify the Product channel when cancel the incident")
 	vars.StringVar(&env.Timezone, "timezone", "America/Sao_Paulo", "The local time of a region or a country used to create a event.")
 	vars.IntVar(&env.SLAHoursToClose, "hellper_sla_hours_to_close", 168, "SLA hours to close")
+	vars.StringVar(&env.MeetingConfig.ProviderName, "hellper_meeting_provider", "matrix", "Name of meeting provider that will create a War room on incident start.")
+	vars.StringVar(&meetingProviderConfig, "hellper_meeting_provider_config", "{}", "Specific config of meeting provider that will create a War room on incident start.")
 
 	vars.Parse()
+
+	env.MeetingConfig.ProviderConfig = getStringMapFromJSON(meetingProviderConfig)
+
 	return env
 }
+
+func getStringMapFromJSON(mapAsJSONString string) map[string]string {
+	stringMap := make(map[string]string)
+
+	err := json.Unmarshal([]byte(mapAsJSONString), &stringMap)
+	if err != nil { // TODO: move this to a logger
+		fmt.Printf("Config with invalid json format. Config: %s", mapAsJSONString)
```

Comment:
Should we do it now or later?

Reply:
We already have a task for reviewing logs, right? I believe we can leave it for this MR!